packages/core/src/components/tag-input/resizeableInput.tsx

```diff
@@ -0,0 +1,50 @@
+/* !
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ */
+
+import React, { forwardRef, useEffect, useRef, useState } from "react";
+
+import { HTMLInputProps } from "../../common";
+
+type ResizeableInputProps = HTMLInputProps;
+export type Ref = HTMLInputElement;
+
+export const ResizeableInput = forwardRef<Ref, ResizeableInputProps>(function ResizeableInput(props, ref) {
```

Comment:
I believe "resizable" is more correct, at least for US English. We use that elsewhere in Blueprint, so please rename all references to "resizeable" -> "resizable" (including the file name).

Reply:
ack